src/interface/polarplot.jl

```diff
@@ -53,11 +54,21 @@ julia> polarplot(range(0, 2π, length = 20), range(0, 2, length = 20))
 
 `Plot`, `lineplot`, `BrailleCanvas`
 """
-function polarplot(θ::AbstractVector, 𝓇::Union{Function,AbstractVector}; kw...)
+function polarplot(
+    θ::AbstractVector,
+    𝓇::Union{Function,AbstractVector};
+    𝓇lim = (0, 0),
```

Comment:
I think I'm going to change my mind about unicode characters like `𝓇` and use ascii instead. Especially since `𝓇lim` is a keyword argument, it might be difficult for user to type / input. I you want you can change `θ` to `theta` and `𝓇` to `rad` or something like that. I you don't have time to change this, I will do that later.

Reply:
Yes, I find it awkward, too.
If you don't mind, I will change 𝓇 to r.
I like `rlim`. It is similiar to `ylim` and `xlim`.
I'm fine with θ, especially because it is not a keyword argument.

Reply:
> If you don't mind, I will change 𝓇 to r. 

Pretty obvious to me that `r` is a radius in this context so :+1:.

Don't forget to change occurrences in the docstring like https://github.com/JuliaPlots/UnicodePlots.jl/blob/23a9f9935125ba388d6ef376e6ca88f083251806/src/interface/polarplot.jl#L5 and in the tests.

Thanks !

Reply:
Done!